backend/router.js

```diff
@@ -5,7 +5,7 @@ const algoTest = require("./Algorithm/algoTest.js");
 const algoController = require("./Algorithm/algo_controller.js")
 
 //define get request for retreiving user information
-router.get("/token/:token/user", Controller.getUser)
+router.get("/users/:id/user", Controller.getUser)
```

Comment:
I forgot but is token and userID the same thing?

Reply:
userID is the spotify ID and token back then was the access token